libs/natparse/src/main/java/org/amshove/natparse/parsing/ParserErrors.java

```diff
@@ -179,7 +179,7 @@ public static ParserDiagnostic noTargetForRedefineFound(RedefinitionNode redefin
 	public static ParserDiagnostic redefinitionLengthIsTooLong(RedefinitionNode node, double redefinitionLength, double maxLength)
 	{
 		return ParserDiagnostic.create(
-			"Length of redefinition (%s bytes) exceeds target length (%s bytes)".formatted(DataFormat.formatLength(redefinitionLength), DataFormat.formatLength(maxLength)),
+			"Length of redefinition (%s bytes) exceeds target length (%s bytes) of %s".formatted(DataFormat.formatLength(redefinitionLength), DataFormat.formatLength(maxLength), node.declaration().source()),
```

Comment:
I like that change, makes it more clear in the diagnostic title :)

Reply:
Great :)